scripts/mk.js

```diff
@@ -12342,19 +12346,19 @@ function accelere(iX, iY, iI, iJ) {
 	return false;
 }
 
-function flowShift(iX,iY, iP) {
+function flowShift(iX,iY) {
 	var lMap = getCurrentLMap(collisionLap);
 	if (lMap.flows) {
 		var oRectangles = lMap.flows.rectangle;
 		for (var i=0;i<oRectangles.length;i++) {
 			var oFlow = oRectangles[i];
-			if (pointInRectangle(iX,iY, oFlow[0]) && (!iP||oFlow[2]))
```

Comment:
This was intended actually, it allows to not be affected by the current in yoshi falls and the bridge in bowser castle (DS) and thus take the shortcuts without issues.
I'm open to change this behavior but is it something that was requested by the community, or you just assumed it was a bug?

Reply:
The request does include the spinning state, so the community seems to agree with this change (9 likes and no dislikes), so the community seems to agree with this change